handler_stream.go

```diff
@@ -161,6 +164,9 @@ func (b *BidiStream[Req, Res]) ResponseTrailer() http.Header {
 // Send a message to the client. The first call to Send also sends the response
 // headers.
 func (b *BidiStream[Req, Res]) Send(msg *Res) error {
+	if msg == nil {
+		return b.conn.Send(nil)
+	}
```

Comment:
Since client-side streams accept `nil` for `Send`, it seemed like the handler-side streams should too.